plugins/webp-uploads/fallback.js

```diff
@@ -100,8 +112,12 @@ window.wpPerfLab = window.wpPerfLab || {};
 
 			if (
 				node.nodeName !== 'IMG' ||
-				( ! node.src.match( /\.webp$/i ) &&
-					! srcset.match( /\.webp\s+/ ) )
```

Comment:
yea, that is probably better. I was wondering about handling fallbacks when users generate one format (WebP) for a while then switch to another format (AVIF). 

Currently the fallback will only handle the currently selected output format - it would be better if it were more flexible and handled both even on the same page. I haven't had a chance to work on that and I'm not sure it is even worth the effort. Ultimately, I feel like the fallback script is never going to be anything but a mediocre solution especially compared to using the `picture` element to implement a fallback.

Reply:
updated in 2266ea81ad344df77438213b6250877a67aa529f